co2eq/purchase/footprints.yml

```diff
@@ -3934,3 +4486,1847 @@ _children:
     unit: EUR
     coicopCode: 07.2.2
     year: 2011
+  CLOTHING AND FOOTWEAR:
```

Comment:
Should we move clothing (coicopCode 03.1) here as well?

Reply:
Not sure what you mean?
Is the google sheet correct? or is the script that I ran to create the output incorrect?